scripts/secnetperf.ps1

```diff
@@ -198,7 +200,20 @@ INSERT OR IGNORE INTO Secnetperf_tests (Secnetperf_test_ID, Kernel_mode, Run_arg
             foreach ($item in $Test.Values[$tcp]) {
                 $SQL += @"
 `nINSERT INTO Secnetperf_test_runs (Secnetperf_test_ID, Secnetperf_commit, Client_environment_ID, Server_environment_ID, Result, Secnetperf_latency_stats_ID)
-VALUES ($TestId, '$MsQuicCommit', $env, $env, $item, NULL);
+VALUES ($TestId-tcp-$tcp, '$MsQuicCommit', $env, $env, $item, NULL);
+"@
+            }
+        }
+
+        if ($Test.Metric.startsWith("latency")) {
```

Comment:
We also need to collect RPS. And what about HPS?

Reply:
I was thinking of a follow up PR, once this integrates well. Also, need to add and persist IO information as well (wsk, epoll...).

Reply:
although, the changes are small enough we could add that in this PR as well.

Reply:
Ok.